src/DataViews/include/DataViews/DataView.h

```diff
@@ -180,6 +196,16 @@ class DataView {
     return nullptr;
   }
 
+  ErrorMessageOr<void> ExportToCsv(const std::string& file_path);
+
+  template <typename T>
+  void ReportErrorIfAny(const ErrorMessageOr<T>& error_message_or,
+                        const std::string& error_window_title) const {
```

Comment:
Nit: I know `SendErrorToUi` takes a string, but we try to get away from `const std::string&` arguments if possible. Could you make this argument a `std::string_view`?

Reply:
This would also have the advantage that if a `std::string` needs to be constructed it will only be constructed in the error case. (That's of course not a good enough argument on its own)